downstream/modules/platform/proc-connecting-nodes-through-mesh-ingress.adoc

```diff
@@ -30,51 +30,56 @@ metadata:
     name:
     namespace:
 spec:
-    deployment name: 
+    deployment_name: 
 ----
 +
 Where:
 
 * *apiVersion*: defines the versioned schema of this representation of an object. 
 Servers should convert recognized schemas to the latest internal value, and might reject unrecognized values.
++
+Set the value to `automationcontroller.ansible.com/v1alpha1`.
 * *kind*: Is the type of node to create. 
++
 Set the value to `AutomationControllerMeshIngress`. 
 `AutomationControllerMeshIngress` controls the deployment and configuration of mesh ingress on {ControllerName}.
-* *name*: is the name of the mesh ingress node.
-* *namespace*: is which Kubernetes namespace to deploy the mesh ingress into.
-This must be in the same namespace as the {ControllerName} that the mesh is connected to
-* *deployment_name*: You can find the deployment name by using:
+* *name*: is the name of the mesh ingress node. You must choose this name TBD
+* *namespace*: is which Kubernetes namespace to deploy the mesh ingress into and is determined by the operator. TBD
+This must be in the same namespace as the {ControllerName} that the mesh is connected to.
+* *deployment_name*: Use the `deployment_name' `aap-controller`.
+Using the following command to determine the `deployment_name`
 +
 ----
-oc get ansible-automation-platform.
+oc get automation_controllers
 ----
 +
-//Additionally you can use:
-+
-//* *external_hostname*: an optional field used for specifying the external hostname defined in an user managed ingress.
-//* *external_ipaddress*: an optional field used for specifying the external IP address defined in an user managed ingress
-//* *ingress_type*: Ingress type for ingress managed by the operator.
-//Where options are:  
-//** none (default) 
-//** Ingress
-//** IngressRouteTCP
-//** Route (default when deployed on OpenShift)
-//* *ingress_api_version*: the API Version for ingress managed by the operator. 
-//This parameter is ignored when `ingress_type=Route`.
-//* *ingress_annotations*: annotation on the ingress managed by the operator
-//* *ingress_class_name*: the name of ingress class to use instead of the cluster default. 
-//This parameter is ignored when `ingress_type=Route`.
-//* *ingress_controller*: special configuration for specific Ingress Controllers. 
-//This parameter is ignored when `ingress_type=Route`.
+always supplies `aap-controller`.
```

Comment:
i don't think this is correct the user will need to provide which instance of AutomationController do they want to attach the mesh ingress that they are creating